config.go

```diff
@@ -10,6 +10,13 @@ import (
 	"github.com/segmentio/parquet-go/compress"
 )
 
+type PageReadMode int
```

Comment:
Let's add a bit of documentation on this type (and the constants as well maybe)?

Reply:
added some comments. let me know if you'd like anything else.